File: rm_duplicates_in_ll.cpp
ListNode* deleteDuplicates(ListNode* head) {
    ListNode* result = head;
    if(head == NULL){
        return head;
    }
    else{
        while(head->next!=NULL){
            if(head->val == head->next->val){
                head->next = head->next->next;
            }
            else{
                head = head->next;
            }
        }
        return result;
    }
}

File: rm_duplicates_in_ll.py
def deleteDuplicates(self, head: Optional[ListNode]) -> Optional[ListNode]:
    if head == None:
        return head
    result = head
    while head.next:
        if head.val == head.next.val:
            head.next = head.next.next
        else:
            head = head.next
    return result
